src/build.js

```diff
@@ -35,48 +35,56 @@ export async function generateBuildMap(configObj) {
             {format: 'module'},
             () => {}
           );
-          buildMap.push([outputFileURL, source]);
+          outputFileList.push([outputFileURL, source]);
         }
       })
     );
 
-    return buildMap;
+    return outputFileList;
   }
   return await iterateDir(configObj.inputDir);
 }
 
-export async function generateBundleOutputObj(buildMap) {
-  const filenames = buildMap.map(buildPair => buildPair[0].pathname);
+export async function generateBundleOutputObj(outputFileList) {
+  const filenames = outputFileList.map(
+    ([inputTreeFileURL]) => inputTreeFileURL.pathname
```

Comment:
For portability, I assume this should be `fileURLToPath(inputTreeFileURL)`?

Reply:
Not sure what exactly you're referring to, `filenames`? or the map function used to get these?

Reply:
It looks like you're using `someURL.pathname` to convert a URL to a filename. Which happens to work for certain files on certain operating systems. But it's not generally correct. E.g. any path with a space will be incorrect on Linux/macOS:

```
> new URL('file:///tmp/a%20b.js').pathname
'/tmp/a%20b.js'
> url.fileURLToPath('file:///tmp/a%20b.js')
'/tmp/a b.js'
```

Reply:
Ah I see, thanks for the tip, updated to use `url.fileURLToPath` now